Example/Messaging/Tests/FIRInstanceIDWithFCMTest.m

```diff
@@ -54,7 +55,11 @@ - (void)tearDown {
 }
 
 - (void)testFCMAutoInitEnabled {
-  FIRMessaging *messaging = [FIRMessaging messagingForTests];
+  NSString *const kFIRMessagingTestsAutoInit = @"com.messaging.test_autoInit";
+  NSUserDefaults *defaults = [[NSUserDefaults alloc] initWithSuiteName:kFIRMessagingTestsAutoInit];
+  FIRMessaging *messaging = [FIRMessagingTestUtilities messagingForTestsWithUserDefaults:defaults];
+  id classMock = OCMClassMock([FIRMessaging class]);
```

Comment:
Please add `[classMock stopMocking]` after this otherwise the `FIRMessaging` class could be mocked for who knows how long (yay OCMock!)